module/documents/actor/actor.mjs

```diff
@@ -815,6 +815,9 @@ export default class Actor5e extends Actor {
       data.skillBonus = Roll.replaceFormulaData(globalBonuses.skill, data);
     }
 
+    // Calculate skill minimum roll
+    const skillMinimum = Math.max(skl?.minimum ?? 0, abl?.minimum?.check ?? 0) || false;
```

Comment:
Probably ok to leave these as `undefined` rather than setting them to `false`. `getMinimumAttackRoll` returns `null`, for example, and that value is passed to `d20Roll` without issue, so using any falsy value is probably fine.